### PR TITLE
Theta

### PR DESCRIPTION
**Pull request by MSmit99**
Attempt 2

### CODE REVIEW

**Review by reddign — COMMENTED** [review]
Thanks for all your documentation notes.

Final merge approved after a rebase with main that I preformed.

File: web_src/general/hangmanSettings.php
Comment (Owy137) [inline web_src/general/hangmanSettings.php:1]: Both of these should be create.php, not add.php?

File: web_src/general/triviaSettings.php [hunk 1]
```diff
@@ -143,7 +143,7 @@ function updateQuestionList() {
             <div id="welcome-text">Trivia Settings</div>
             <div id= "basicContainer">
             
-            <form action="../../data_src/api/question/add.php" method="post" id="add-question" style="text-align: left">
```
Comment (Owy137) [inline web_src/general/triviaSettings.php:146]: trivia folder was renamed to question, keep what is on main branch

File: web_src/general/register.php [hunk 2]
```diff
@@ -16,7 +16,7 @@
         <div id="basicContainer">
             
         <!-- Registration Form -->
-        <form action="../../data_src/api/admin/create.php" method="post" style= "text-align: left">
+        <form action="../../data_src/api/admin/add.php" method="post" style= "text-align: left">
```
Comment (Owy137) [inline web_src/general/register.php]: These should be changed back to create.php

File: web_src/trivia/trivia.js [hunk 3]
```diff
@@ -33,7 +33,7 @@ function displayRandomQuestion() {
 }
 
 function randomQuestion() { // Get questions from database
-    fetch('../../data_src/api/question/questions.php', {method: 'get'}) // TODO: Change file path for FTP
+    fetch('../../data_src/api/trivia/questions.php', {method: 'get'}) // TODO: Change file path for FTP
```
Comment (Owy137) [inline web_src/trivia/trivia.js]: trivia was renamed to question, keep main branch

**Review by Owy137 — CHANGES_REQUESTED** [review]

**Review by Owy137 — CHANGES_REQUESTED** [review]
Made updates to Theta branch side of merge

**Review by Owy137 — CHANGES_REQUESTED** [review]
Fixed reference to add.php

**Pull request merged**